lib/screens/productsNew/newUI.dart

```diff
@@ -329,13 +332,16 @@ class _ProductsUINewState extends State<ProductsUINew> {
                                     });
                                     switch (index) {
                                       case 0:
-                                        apiBloc.add(GetVegetables());
+                                        apiBloc.add(GetProducts(
+                                            type: catArray[index].resname));
                                         break;
```

Comment:
No point in having multiple cases here as index will be able to handle it ?

Reply:
yea will remove them